tests/api_test/api_test.cpp

```diff
@@ -1001,8 +1001,6 @@ test_sock_addr_native_program_load_attach(const char* file_name)
         test_sock_addr_native_program_load_attach((const char*)"cgroup_sock_addr2_"##version ".sys"); \
     }
 
-DECLARE_REGRESSION_TEST_CASE("0.9.0")
```

Comment:
Match https://github.com/microsoft/ebpf-for-windows/pull/2829/files#diff-c336aef1a5b6b610ede7ec16ac04f798e3620811eb0416754a40f2c741e4ece2R1004